charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,106 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team is:
+
+* Working with data contributors who are generating new data types to develop a deep understanding of their data
+* Assisting data contributors to describe their experiments in a manner that supports findability, reusability and reproducability within the DCP ecosystem
+* Helping data contributors to submit their data to the DCP, resolving any emerging issues during the process
+
+## Definitions
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* -  the Ingest Support team member who assists labs contributing data to the HCA.
```

Comment:
In Ingest Service, @tburdett defines:

_Data wrangler - a data contributor who works as part of the HCA consortium to provide assistance to biologists wanting to upload data into the DCP_

Can we align one definition across charters?